API/Parser/Parser.cs

```diff
@@ -64,43 +64,43 @@ public static class Parser
         /// <summary>
         /// Recognizes the Special token only
         /// </summary>
-        private static readonly Regex SpecialTokenRegex = new Regex(@"SP\d+",
+        private static readonly Regex SpecialTokenRegex = new Regex(@"(SP|HS)\d+",
             MatchOptions, RegexTimeout);
 
+        private const string seriesExpr = $@"(?<Series>{balanced}?)";
```

Comment:
You need to use proper naming convention. Constant variables need to start with a capital letter in C#.

Reply:
Done!